CEGameApp.API/Dtos/UserForRegisterDto.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace CEGameApp.API.Dtos
+{
+    public class UserForRegisterDto
+    {
+        [Required]
```

Comment:
so I see you have validation attributes on these properties but you are not doing anything with them.  you need to call Model.IsValid(model) in your controller to actually check against these attributes.

Reply:
This will be done.  It's there to remind me to make use of it.